pkg/backend/display/progress.go

```diff
@@ -857,7 +858,6 @@ func (display *ProgressDisplay) processNormalEvent(event engine.Event) {
 		// once we start hearing about actual resource events.
 
 		payload := event.Payload.(engine.PreludeEventPayload)
-		display.isPreview = payload.IsPreview
```

Comment:
this previous form meant you could observe a column header changing from "Status" to "Plan".  There's no need for this given that all callers into the display know if we're in preview mode or not.